modules/profile-store/pages/dotnet.adoc

```diff
@@ -529,7 +529,9 @@ image:00503-remote-cluster.png[Remote Cluster]
 
 image:00504-add-rep.png[Add replication]
 
-Note that we have checked the option *Enable advanced filtering*. It will allow you to filter which documents should be replicated by applying a regex on the document id. In the case above, only documents with ids starting with *"user::"* will be replicated. As in our User Profile store, only the *User* has this prefix, it will be the only document type which will be replicated to other clusters. You can read more about it link:https://docs.couchbase.com/server/6.0/learn/clusters-and-availability/xdcr-filtering.html[here].
+Note that we have checked the option *Enable advanced filtering*. It will allow you to filter which documents should be replicated. Since Couchbase 6.5 you can filter documents using multiple attributes of your document using a syntax very similar to N1QL. You can read more about it link:https://docs-staging.couchbase.com/server/6.5/xdcr-reference/xdcr-filtering-expressions.html[here]
+
+In the case above, only documents with the attribute *_class* equals to *com.cb.demo.userProfile.model.UserEntity* will be replicated. As in our User Profile store, only the *UserEntity* has this prefix, it will be the only document type which will be replicated to other clusters.
```

Comment:
Are you sure this applies to the dotnet tutorial? `_class` looks Spring specific